robots/indexpagecreator/update-jobs-with-latest-indexpagecreator-image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+
+docker pull kubevirtci/indexpagecreator
+sha_id=$(docker images --digests kubevirtci/indexpagecreator | grep 'latest ' | head -1 | awk '{ print $3 }')
+
+for file in $(grep -l 'image: .*/kubevirtci/indexpagecreator' ../../github/ci/prow/files/jobs/**/**/*-periodics.yaml); do
+    sed -i -E 's/index\.docker\.io\/kubevirtci\/indexpagecreator@sha256\:[a-z0-9]+/'"index\.docker\.io\/kubevirtci\/indexpagecreator@$sha_id"'/g' \
```

Comment:
Slight (pedantic) improvement: `sed -i "s#index\.docker\.io/kubevirtci/indexpagecreator@sha256\:[a-f0-9]\+#index.docker.io/kubevirtci/indexpagecreator@${sha_id}#g" $file`

Reply:
Good one! Must admit that I'm still sad I never got to use `yq` here :-/